src/scss/components/_primary-sidebar.scss

```diff
@@ -0,0 +1,10 @@
+.cmp-primary-sidebar {
+  background-color: var(--gray-background-color);
+  padding: 1rem;
+  margin-top: 1.5rem;
+  
+  @media (min-width: 50rem) {
+    margin: 0 0 0 3.75rem;
+    width: 18.75rem;
```

Comment:
Rather than setting margins and width, I think we can take advantage of flexbox features a little more.

In `.cmp-post-body`, you could use `gap: 1.5rem 3.75rem;` to achieve what the margins are doing, and then in `.cmp-primary-sidebar`, you could use `flex-basis` instead of `width` to set the ideal size while still allowing it to grow or shrink as needed.